pkg/operator/ceph/csi/cluster_config.go

```diff
@@ -149,6 +149,9 @@ func updateCsiClusterConfig(curr, clusterKey string, newCsiClusterConfigEntry *C
 			if newCsiClusterConfigEntry.RBD.RadosNamespace != "" || newCsiClusterConfigEntry.RBD.NetNamespaceFilePath != "" {
 				centry.RBD = newCsiClusterConfigEntry.RBD
 			}
+			if len(newCsiClusterConfigEntry.ReadAffinity.CrushLocationLabels) != 0 {
```

Comment:
this all is duplicate can we refactor this to a common place maybe with another PR

Reply:
the current implementation of `updateCsiClusterConfig` is like that.
We can refactor the function in another PR.